python/tests/kernel/test_kernel_features.py

```diff
@@ -1995,6 +1995,31 @@ def invalid_unsupported():
     with pytest.raises(RuntimeError):
         cudaq.sample(invalid_unsupported)
 
+def test_in_comparator():
```

Comment:
nit: Could we also throw in a simple `not in`  test here?

Reply:
Thank you for catching that, I'll add it right now